lib/cartodb/api/request.rb

```diff
@@ -61,9 +63,9 @@ def make_request(method, params = nil, headers = nil, body = nil)
         parse(response.body)
       end
 
-      def configure_request(request, params = nil, headers = nil, body = nil)
+      def configure_request(request, params: nil, headers: nil, body: nil, payload: nil)
         request.params.merge!(params) if params
-        request.headers['Content-Type'] = 'application/json'
+        request.headers['Content-Type'] = 'application/json' unless payload
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.